libshaderc/src/shaderc_cpp_test.cc

```diff
@@ -492,22 +495,22 @@ TEST_F(CppInterface, ErrorTagIsInputFileName) {
 
 TEST_F(CppInterface, PreprocessingOnlyOption) {
   options_.SetPreprocessingOnlyMode();
-  shaderc::CompilationResult result = compiler_.CompileGlslToSpv(
+  const shaderc::AssemblyCompilationResult result = compiler_.PreprocessGlsl(
```

Comment:
I think the type should be "PreprocessedSourceCompilationResult", not "AssemblyCompilationResult".